packages/patternfly-4/react-core/src/components/EmptyState/EmptyStatePrimary.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import { css } from "@patternfly/react-styles";
+import styles from "@patternfly/react-styles/css/components/EmptyState/empty-state";
+
+export interface EmptyStatePrimaryProps
+  extends React.HTMLProps<HTMLDivElement> {
+  /** Additional classes added to the EmptyStatePrimary */
+  className?: string;
+  /** Content rendered inside the EmptyStatePrimary */
+  children: React.ReactNode;
+}
+
+export const EmptyStatePrimary: React.FunctionComponent<
+  EmptyStatePrimaryProps
+> = ({ children, className = '', ...props }: EmptyStatePrimaryProps) => {
+  return (
+    <div className={css(styles.emptyStatePrimary, className)} {...props}>
```

Comment:
Hmm, when I put `<EmptyStatePrimary className="custom-empty-state-prim-cls" id="empty-state-prim-id">
<Button variant="link">Link</Button></EmptyStatePrimary>` in one of the empty state component examples, I get `ReferenceError: EmptyStatePrimary is not defined`. Do you know what I'm doing wrong?

Reply:
No, I think this is my mistake. I forgot to export it. I will fix that. Thank you!